packages/worker/src/utilities/email.ts

```diff
@@ -162,6 +164,47 @@ export async function sendEmail(
       contents: opts?.contents,
     }),
   }
+  if (opts?.attachments) {
+    const baseUrl = appContext.getPlatformURL()
+    const attachments = await Promise.all(
+      opts.attachments?.map(async attachment => {
+        const isFullyFormedUrl =
+          attachment.url.startsWith("http://") ||
+          attachment.url.startsWith("https://")
+
+        const url = isFullyFormedUrl ? attachment.url : baseUrl + attachment.url
+        let headers
+
+        if (env.isTest()) {
+          headers = {
+            Host: "minio-service",
```

Comment:
Not very nice, and kind of hacky, but seems to be the only way to get around the minio host requirement.

Reply:
Mm, I'm not the world's biggest fan of test specific code mixed in with production code. I won't block on it, but is this our only option?

Reply:
Me and Peter took a look into this last week - we tried to find a way to remove the host setting when running in tests, but sadly its unavoidable. In production we'll get away with this as the request will go through the proxy, rather than directly to the service.

There were two other ways this could be solved:
1. Run the budibase/proxy container as well in our test suites to connect through, this would be the most correct to our production environments, but also a little un-necessarily complex.
2. Mocking the request in someway as part of the test, to make sure that the header is attached as expected as part of the test.